server/asset/common.go

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2019, The Decred developers
+// See LICENSE for details.
+
+package asset
+
+type Error string
+
+func (err Error) Error() string { return string(err) }
+
+const (
+	UnsupportedScriptError = Error("unsupported script type")
```

Comment:
I'm not sure if we should have a set of constant errors like this or not.

Reply:
Since it allows checking for errors directly (not by the error string), I'm good with this.

Reply:
Actually, this seems apropos: https://github.com/decred/dcrwallet/pull/1539

Reply:
Just something to plan for that is since we're still sort of supporting go 1.12 even though dcrdex is a long way from complete.